data/migrations/20200207181330_project-schema.js

```diff
@@ -0,0 +1,60 @@
+exports.up = function(knex) {
+  return knex.schema
+    .createTable("projects", project => {
+      project.increments();
+      project
+        .text("project_name")
+        .notNullable()
+        .unique();
+      project.text("description");
+      project.boolean("completed");
```

Comment:
a boolean that indicates if the project has been completed. This column cannot be NULL, the default value should be false.